src/xcode/ENA/ENA/Source/Services/HealthCertificateValidation/__tests__/HealthCertificateValidationServiceValidationTests.swift

```diff
@@ -427,7 +427,7 @@ class HealthCertificateValidationServiceValidationTests: XCTestCase {
 			XCTFail("report must not be nil")
 			return
 		}
-		XCTAssertEqual(error, .TECHNICAL_VALIDATION_FAILED)
+		XCTAssertEqual(error, .TECHNICAL_VALIDATION_FAILED(expirationDate: healthCertificate.expirationDate, signatureInvalid: true))
```

Comment:
Shouldn't there be tests for invalid signature but valid date and vice versa?

Reply:
Done